modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/AbstractResourceManager.java

```diff
@@ -67,9 +67,13 @@ public static JSONDecoder getDecoder() throws CharonException {
      * @return endpoint URL
      * @throws NotFoundException
      */
-    public static String getResourceEndpointURL(String resource) throws NotFoundException {
+    public static String getResourceEndpointURL(String resource, String context) throws NotFoundException {
```

Comment:
This is an API change. Better to overload this method to ensure backward compatibility.